src/components/text/text.module.css

```diff
@@ -101,31 +135,36 @@
   line-height: 128.571%;
   letter-spacing: 0.00875rem;
 
-  @media (max-width: 375px) {
+  @media (max-width: 425px) {
     font-size: 0.75px;
```

Comment:
What is this? Text for ants??

![Screenshot 2024-12-09 at 15 33 56](https://github.com/user-attachments/assets/87979dff-c466-40ae-aab3-47b8d6c09405)

Reply:
```suggestion
    font-size: 0.75rem;
```

Reply:
hahaha, good catch 🐜